### PR TITLE
[Android] sync to cs-api update

### PR DESCRIPTION
**Pull request by jaeyun-jung**
1. change method name to set/get the property
2. handle empty node name
3. define method to get model file in testcase

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

**Comment by jaeyun-jung** [comment]
Waiting for ACR process, on API review.

File: api/android/api/src/main/jni/nnstreamer-native-api.c [hunk 1]
```diff
@@ -30,7 +30,6 @@ GST_PLUGIN_STATIC_DECLARE (amcsrc);
 extern void init_filter_cpp (void);
 extern void init_filter_custom (void);
 extern void init_filter_custom_easy (void);
-extern void init_filter_tflite (void);
```
Comment (myungjoo) [inline api/android/api/src/main/jni/nnstreamer-native-api.c]: This should be optionally included/excluded by build options along with nn-runtime, snap, and snpe(in the future). The clients want to have multiple versions of .aar depending on the requirements of each application.

Or are you going to make separate .so for each "optional" subplugins in .aar?

Comment (jaeyun-jung) [reply api/android/api/src/main/jni/nnstreamer-native-api.c]: I added another PR, build option for subplugins.

Comment (myungjoo) [reply api/android/api/src/main/jni/nnstreamer-native-api.c]: If that's the case, this is good for now.

**Pull request merged**